3Erronka/src/Controlador/Main.java

```diff
@@ -0,0 +1,62 @@
+package Controlador;
+
+import Vista.VLoginPantaila;
+import Vista.VOngiEtorria;
+import Vista.VZinemaAukeraketa;
+import DatuBasea.PelikulaDAO;
+import DatuBasea.ZinemaDAO;
+import Modelo.Zinema;
+
+public class Main {
+
+	public static void main(String[] args) {
+		PelikulaDAO filmak = new PelikulaDAO();
+		Zinema[] zinemak = new Zinema[4];
+		ZinemaDAO zinem = new ZinemaDAO();
```

Comment:
Etorkizunean 10 zinema baduzue, 10 lerro sartuko dituzue? Egokiagoa for bat agian?

Reply:
Konponduta. Funtzio bat gehitu egin dut ZinemaDAO-n non zenbatzen ditu zenbat zinema dauden datu basean eta zenbaki horrekin array bat sortzen du. ZinemaDAOTest-en test berri bat gehitu dut funtzio horretaz